packages/@aws-cdk/interfaces/lib/lambda-layer-asset.ts

```diff
@@ -0,0 +1,16 @@
+import { IConstruct } from 'constructs';
+
+/**
+ * A Construct that can be used as the asset for defining a Lambda Layer
+ */
+export interface ILambdaLayerAsset extends IConstruct {
```

Comment:
Don't we need a `bind()` in here as well in case the asset lives on disk?

Reply:
In fact the actual asset package is going to have to return something like `AssetOptions`... it can't go `new Asset()` because that's in core and core needs to depend on this.